### DISCUSSION TITLE
getting blocked while scraping linkedin profiles

### DISCUSSION THREAD

**Discussion by Sajidch1** [ultrafunkamsterdam/undetected-chromedriver#1773] [Q&A]
I'm attempting to scrape LinkedIn profiles using undetected_chromedriver, but LinkedIn still detects and temporarily blocks my account. How can I resolve this issue? Your guidance would be appreciated.

**Comment by Leoplazavzla** [comment]
Did you find a solution? 
I'm using mozilla webdriver but I get detected after 9 minutes.

**Comment by C-Nubela** [comment]
> I'm attempting to scrape LinkedIn profiles using undetected_chromedriver, but LinkedIn still detects and temporarily blocks my account. How can I resolve this issue? Your guidance would be appreciated.

Hey there @Sajidch1 -- I work at Proxycurl, a B2B data provider, and if you're looking to scrape/pull LinkedIn profile without headaches, [our API here might interest you](https://nubela.co/proxycurl/linkedin?utm_campaign=content_amplification&utm_source=github&utm_medium=social&utm_content=comments).

Best of luck!
